providers/windows/process_windows.go

```diff
@@ -304,17 +304,17 @@ func (p *process) User() (types.UserInfo, error) {
 		return types.UserInfo{}, errors.Wrap(err, "GetTokenUser failed")
 	}
 
-	sid, err := tokenUser.User.Sid.String()
-	if err != nil {
+	sid := tokenUser.User.Sid.String()
+	if sid == "" {
 		return types.UserInfo{}, errors.Wrap(err, "failed to look up user SID")
```

Comment:
It no longer makes sense to call `errors.Wrap(err, ...)` here (or in the change below). Can you please change them to `errors.New("...")`?

Reply:
o right, didn't realise that line was just using the std pkg errors.